product_attribute_model_link/migrations/16.0.1.0.2/pre-migrate.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (C) 2024 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+def migrate(cr, version):
+    # Create temporary columns to store data before the field becomes computed
+    cr.execute(
+        """
+        ALTER TABLE product_attribute_value
```

Comment:
Just a question: can't we create columns with the final names right from the start? Without using those intermediate ones.

Reply:
Thanks for the feedback! I've adjusted the script to use the final column names from the start, simplifying the migration process. Please review it when you have a moment.